distributions/distribution-resources/src/main/resources/conf/items/readme.txt

```diff
@@ -0,0 +1,7 @@
+
+openHAB2 conf/items folder
+
+Your item definitions go here.
+All items files have to have the ".items" file extension and must follow a special syntax.
+
+Check out the openHAB Wiki for more details: https://github.com/openhab/openhab/wiki/Explanation-of-items
```

Comment:
Here I would prefer a link to docs.openhab.org as well. I am aware that there is no page with this content yet, but again, this could be a good moment to transfer the content from the openHAB 1 wiki. How about creating a new page http://docs.openhab.org/configuration/items.html (with more or less the content of the wiki page).